### DISCUSSION TITLE
Need a stylist-rs discord server

### DISCUSSION THREAD

**Discussion by Madoshakalaka** [futursolo/stylist-rs#46] [General]
I love stylist <3

**Comment by WorldSEnder** [comment]
[Created one](https://discord.gg/4T3kfMbQqc) but ultimately your call @futursolo

**Comment by futursolo** [comment]
Thank you for purposing a discord server.

Whilst I think it would be a good idea for an active community, it might be "too early" for stylist to have an official chatroom. (Not enough active users?) Nor do I have time to moderate it.

In the meantime, I have enabled the discussion board of this repository, should you have any questions with stylist, you may ask there and I will answer them to my best ability.

With that being said though, I don't mind endorsing an unofficial discord room in the README if someone is willing to moderate it.

**Comment by WorldSEnder** [comment]
I setup roles and we could only enable maintainers/contributors to talk for now, but I'm fine with the Discussions board on gh.
